cocos/animation/event/event-emitter.ts

```diff
@@ -0,0 +1,24 @@
+import { Node } from '../../scene-graph';
+
+/**
+ * Invokes methods with specified `methodName` on each component of `node`,
+ * the methods were registered binding to some animation related events(animation clip frame event or animation graph event).
+ *
+ * @param node The node to search components with.
+ * @param methodName The method name to invoke.
+ * @param args The arguments passed to the method.
+ */
+export function invokeComponentMethodsEngagedInAnimationEvent (node: Node, methodName: string, ...args: unknown[]) {
+    const components = node.components;
+    const nComponents = components.length;
+    for (let iComponent = 0; iComponent < nComponents; ++iComponent) {
+        const component = components[iComponent];
+        if (!component.enabled) {
```

Comment:
In our competitor's similar feature,  it does not check for the component enable/disable status. In addition, since our embedded player does not check for the component status as well, i dont think we need to break the compatibility with this change.